packages/remix-deno/globals.ts

```diff
@@ -0,0 +1,12 @@
+/*
+Remix provides `process.env.NODE_ENV` at compile time.
+Declare types for `process` here so that they are available in Deno.
+*/
+
+interface ProcessEnv {
```

Comment:
I think these should all be inside a `declare global`, right?

Reply:
No, `declare global` is only valid syntax in a typescript module (a TS file with `import` / `export` in it).
TS files that are not module (like this one) are automatically in the global scope.